### PR TITLE
Improve search and show detail page styling

### PR DESCRIPTION
**Pull request by florimondmanca**
# Description

- Improve design of landing page (search)

screenshot 2018-10-30 at 16 21 46

- Create more compact layout for show detail page

screenshot 2018-10-30 at 16 22 02

- Style single-page forms (login, signup, profile)

screenshot 2018-11-06 at 09 18 46

- Display a kind message if a search has no results

screenshot 2018-11-06 at 09 18 55

### CODE REVIEW

File: series/static/series/search_series.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+form {
```
Comment (CecileSerene) [inline series/static/series/search_series.css]: I dont know where this is defined, but I dont like the very large input fields for login/signup/profile...

Comment (CecileSerene) [reply series/static/series/search_series.css]: Same with update button on profile

**Comment by florimondmanca** [comment]
[Prettify the search page](https://trello.com/c/V1gweJV7/83-prettify-the-search-page)

File: series/static/series/search_series.css (hunk 1 above)
Comment (florimondmanca) [reply series/static/series/search_series.css]: Fixed

**Pull request merged**